docs/how-to/spacetimedb-standalone.md

```diff
@@ -0,0 +1,250 @@
+
+```nginx
+server {
+    listen 80;
+    server_name example.com;
+
+    location / {
+        proxy_pass http://localhost:3000;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        proxy_set_header Host $host;
+    }
+
+    # This restricts who can publish new databases to your SpacetimeDB instance. We recommend
+    # restricting this ability to local connections. You should uncomment this before you use a
+    # SpacetimeDB server in production.
+    # location /v1/publish {
+        # allow 127.0.0.1;
+        # deny all;
+        # proxy_pass http://localhost:3000;
+        # proxy_http_version 1.1;
+        # proxy_set_header Upgrade $http_upgrade;
+        # proxy_set_header Connection "Upgrade";
+        # proxy_set_header Host $host;
+    # }
+}
+```
+
+This configuration contains a restriction to the `/v1/publish` route. This restriction makes it so that you can only publish to the database if you're publishing from a local connection on the host. You should uncomment that section and then run `sudo systemctl nginx restart` in order to apply that config change.
+
+Enable the configuration:
+
+```sh
+sudo ln -s /etc/nginx/sites-available/spacetimedb /etc/nginx/sites-enabled/
+```
+
+Restart Nginx:
+
+```sh
+sudo systemctl restart nginx
+```
+
+### Configure Firewall
+Ensure your firewall allows HTTPS traffic:
+
+```sh
+sudo ufw allow 'Nginx Full'
+sudo ufw reload
+```
+
+## Step 4: Secure with Let's Encrypt
+
+### Install Certbot
+
+```sh
+sudo apt install certbot python3-certbot-nginx -y
+```
+
+### Obtain an SSL Certificate
+
+Run this command to request a new SSL cert from Let's Encrypt. Remember to replace `example.com` with your own domain:
+
+```sh
+sudo certbot --nginx -d example.com
+```
+
+Certbot will automatically configure SSL for Nginx. Restart Nginx to apply changes:
+
+```sh
+sudo systemctl restart nginx
+```
+
+### Auto-Renew SSL Certificates
+Certbot automatically installs a renewal timer. Verify that it is active:
+
+```sh
+sudo systemctl status certbot.timer
+```
+
+## Step 5: Verify Installation
+
+On your local machine, add this new server to your CLI config. Make sure to replace `example.com` with your own domain:
+
+```bash
+spacetime server add self-hosted --url https://example.com
+```
+
+By default you should be able to publish to this server from your local machine:
+
+```bash
+spacetime publish -s self-hosted <module-name>
+```
+
+If you have uncommented the `/v1/publish` restriction in Step 3 then you won't be able to publish to this instance unless you copy your module to the host first and then publish. We recommend something like this:
+
+```bash
+spacetime build
+scp target/wasm32-unknown-unknown/release/spacetime_module.wasm ubuntu@<host>:/home/ubuntu/
+ssh ubuntu@<host> spacetime publish -s local --bin-path spacetime_module.wasm <module-name>
+```
```

Comment:
Again, I would position the restricted mode as the default.

Reply:
I agree, updated 👍